src/utils.ts

```diff
@@ -1,3 +1,7 @@
+import type { UiSchema } from '@rjsf/core';
+
+const SEPARATOR_ID_RJSF = '_';
```

Comment:
For maintainability purpose, I think this constant should be passed to the `idSeparator` prop of our global `Form`. In this way, we have the guarantee that this constant is in line with your the RJSF Form settings. We could declare it in a `settings.ts` file or in a `.env` file.

Furthermore I would renamed this constant `RJSF_ID_SEPARATOR`. I find it more readable, in this way all constant relying on react json schema form will be prefixed by `RJSF`

Reply:
Ohhh good tips !! I'll go with it thanks !